synapse/util/caches/__init__.py

```diff
@@ -15,27 +15,16 @@
 # limitations under the License.
 
 import logging
-import os
 from typing import Dict
 
 import six
 from six.moves import intern
 
-from prometheus_client.core import REGISTRY, Gauge, GaugeMetricFamily
+from prometheus_client.core import Gauge
 
-logger = logging.getLogger(__name__)
-
-CACHE_SIZE_FACTOR = float(os.environ.get("SYNAPSE_CACHE_FACTOR", 0.5))
-
-
-def get_cache_factor_for(cache_name):
-    env_var = "SYNAPSE_CACHE_FACTOR_" + cache_name.upper()
-    factor = os.environ.get(env_var)
-    if factor:
-        return float(factor)
-
-    return CACHE_SIZE_FACTOR
```

Comment:
Do we not want to keep support for configuring via env vars?

Reply:
Reintegrated that, I forgot about it originally.